opentelemetry-distro/setup.cfg

```diff
@@ -0,0 +1,59 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[metadata]
+name = opentelemetry-distro
+description = OpenTelemetry Python Distro
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = OpenTelemetry Authors
+author_email = cncf-opentelemetry-contributors@lists.cncf.io
+url = https://github.com/open-telemetry/opentelemetry-python/tree/master/opentelemetry-distro
+platforms = any
+license = Apache-2.0
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+python_requires = >=3.5
+package_dir=
+    =src
+packages=find_namespace:
+zip_safe = False
+include_package_data = True
+install_requires =
+    opentelemetry-api == 0.17.dev0
+    opentelemetry-sdk == 0.17.dev0
+
+[options.packages.find]
+where = src
+
+[options.entry_points]
+opentelemetry_distro =
+    distro = opentelemetry.distro:OpenTelemetryDistro
```

Comment:
This is the default Opentelemetry distro right? What would other distros be called? 
`azure = opentelemetry.distro:AzureDistro` something like this?

Reply:
i'd originally called it `otel_distro`, but based on @ocelotl's feedback changed it to `distro`. I don't feel strongly one way or the other on this. I would expect the distro to match the vendor, as you suggested, `azure` would identify the Azure distro.

Reply:
I personally think it's best to prefix everything with an "opentelemetry" identifier just in case there are other libraries using a similar name within the entry points.